### DISCUSSION TITLE
Cannot “open” read-only relationship entries

### DISCUSSION THREAD

**Discussion by AlessioGr** [payloadcms/payload#1685] [Feature Requests & Ideas]
![image](https://user-images.githubusercontent.com/70709113/208157074-8e751e49-82bd-4ed8-bcae-cfba94f28942.png)
I can edit the above relationships (with that I mean the new edit modal next to the remove button).
However, I cannot click that button for the bottom, read-only relationships.

Now even though it's read-only, I expect to be able to click that button, as it doesn't modify the relationship itself, but only the referenced document.

**Comment by jacobsfletch** [comment]
@AlessioGr good call! This should be pretty easy to do. We'll slate this into the next iteration of drawer UI: https://github.com/payloadcms/payload/pull/1668
